sky/compositor/clip_path_layer.cc

```diff
@@ -13,10 +13,12 @@ ClipPathLayer::ClipPathLayer() {
 ClipPathLayer::~ClipPathLayer() {
 }
 
-void ClipPathLayer::Paint(GrContext* context, SkCanvas* canvas) {
+void ClipPathLayer::Paint(PictureRasterzier& rasterizer,
+                          GrContext* context,
```

Comment:
I wonder if we should make an object that has a rasterizer and a context so that we can just pass that object through all these paint calls without having to edit them every time.  For example, a compositor::PaintContext object.